src/assets/scss/_give.scss

```diff
@@ -6,6 +6,10 @@ $colorGrayscale-rock: #b9b9b9;
 $colorGrayscale-slate: #909b9c;
 $colorCru-border: #ddd;
 
+.branded-checkout {
+  text-align: left;
```

Comment:
I think this will be too broad.

Reply:
Do we want to allow sites to change the alignment of the headings and field labels?

Reply:
They should be able to change pretty much anything.